### PR TITLE
Update env var notes to capture Curtis' release notes

### PR DESCRIPTION
**Pull request by samuel-williams-da**
Intended to do this in the last PR, but merged by accident :(

### CODE REVIEW

File: docs/2.10.0/docs/tools/assistant.rst [hunk 1]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
 Environment Variable Interpolation
 ==================================
 Both the ``daml.yaml`` and ``multi-package.yaml`` config files support environment variable interpolation on all string fields.
-Interpolation takes the form of ``${MY_ENVIRONMENT_VARIABLE}`` which are replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
-calling shell.
-A simple example is given below:
+Interpolation takes the familiar form of ``${MY_ENVIRONMENT_VARIABLE}`` which are replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
+calling shell. These can be escaped and placed within strings as per usual environment variable interpolation semantics.
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:214]:
```suggestion
calling shell. These can be escaped and placed within strings according to the environment variable interpolation semantics.
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 2]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+
+This allows you to extract common data, such as the sdk-version, package-name or package-version to outside of said packages ``daml.yaml``, for example
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:216]:
```suggestion
This allows you to extract common data, such as the sdk-version, package-name, or package-version outside of a package's ``daml.yaml``. For example,
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 3]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+in an ``.envrc`` file, or provided by a build system. This feature can also be used for specifying dependency DARs, which would allow you to either store
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:217]:
```suggestion
you can use an ``.envrc`` file or have these values provided by a build system. This feature can also be used for specifying dependency DARs, enabling you to either store
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 4]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+your DARs in a common folder and pass its directory as a variable, shortening the paths in your ``daml.yaml``, or for each dependency to be passed as a
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:218]:
```suggestion
your DARs in a common folder and pass its directory as a variable, shortening the paths in your ``daml.yaml``, or pass each dependency as a
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 5]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+separate variable by an external build system, which may keep them in a temporary cache.
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:219]:
```suggestion
separate variable through an external build system, which may store them in a temporary cache.
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 6]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+
+An example showcasing this is given below:
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:221]:
```suggestion
The following example showcases this:
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 7]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+
+Escape syntax uses the familiar ``\`` prefix: ``\${NOT_INTERPOLATED}``, and interpolation can be disallowed for a config file
 by setting the ``environment-variable-interpolation`` field to ``false``.
 
 .. code-block:: yaml
 
   name: ${NOT_INTERPOLATED}
   environment-variable-interpolation: false
+
+Note that environment variables are case sensitive, meaning ``${MY_VAR}`` and ``${My_Var}`` will not reference the same variable.
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:245]:
```suggestion
Note that environment variables are case sensitive, meaning ``${MY_VAR}`` and ``${My_Var}`` do not reference the same variable.
```

File: docs/2.10.0/docs/tools/assistant.rst [hunk 8]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+  version: ${PROJECT_VERSION}
+  dependencies:
+    // Using a common directory
+    ${DEPENDENCY_DIRECTORY}/my-dependency-1.0.0.dar
+    ${DEPENDENCY_DIRECTORY}/my-other-dependency-1.0.0.dar
+    // Passed directly by a build system
+    ${DAML_FINANCE_DAR}
+    ${MY_DEPENDENCY_DAR}
+
+Escape syntax uses the familiar ``\`` prefix: ``\${NOT_INTERPOLATED}``, and interpolation can be disallowed for a config file
```
Comment (cathyjung-da) [inline docs/2.10.0/docs/tools/assistant.rst:237]:
```suggestion
Escape syntax uses the ``\`` prefix: ``\${NOT_INTERPOLATED}``, and interpolation can be disallowed for a config file
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 9]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
 Environment Variable Interpolation
 ==================================
 Both the ``daml.yaml`` and ``multi-package.yaml`` config files support environment variable interpolation on all string fields.
-Interpolation takes the form of ``${MY_ENVIRONMENT_VARIABLE}`` which are replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
-calling shell.
-A simple example is given below:
+Interpolation takes the familiar form of ``${MY_ENVIRONMENT_VARIABLE}`` which are replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:213]:
```suggestion
Interpolation takes the form of ``${MY_ENVIRONMENT_VARIABLE}``, which is replaced with the content of ``MY_ENVIRONMENT_VARIABLE`` from the
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 10]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+calling shell. These can be escaped and placed within strings as per usual environment variable interpolation semantics.
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:214]:
```suggestion
calling shell. These can be escaped and placed within strings according to the environment variable interpolation semantics.
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 11]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+
+This allows you to extract common data, such as the sdk-version, package-name or package-version to outside of said packages ``daml.yaml``, for example
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:216]:
```suggestion
This allows you to extract common data, such as the sdk-version, package-name, or package-version outside of a package's ``daml.yaml``. For example,
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 12]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+in an ``.envrc`` file, or provided by a build system. This feature can also be used for specifying dependency DARs, which would allow you to either store
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:217]:
```suggestion
you can use an ``.envrc`` file or have these values provided by a build system. This feature can also be used for specifying dependency DARs, enabling you to either store
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 13]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+your DARs in a common folder and pass its directory as a variable, shortening the paths in your ``daml.yaml``, or for each dependency to be passed as a
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:218]:
```suggestion
your DARs in a common folder and pass its directory as a variable, shortening the paths in your ``daml.yaml``, or pass each dependency as a
```

File: docs/2.9.3/docs/tools/assistant.rst [hunk 14]
```diff
@@ -210,21 +210,36 @@ The ``multi-package.yaml`` file supports `Environment Variable Interpolation <#e
...
+separate variable by an external build system, which may keep them in a temporary cache.
+
+An example showcasing this is given below:
```
Comment (cathyjung-da) [inline docs/2.9.3/docs/tools/assistant.rst:221]:
```suggestion
The following example showcases this:
```

**Review by cathyjung-da — APPROVED** [review]
Hi @samuel-williams-da, I completed the PR review as Carrie is OOO. Thank you.

**Comment by samuel-williams-da** [comment]
Thankyou for the thorough review @cathyjung-da , and for replicating across the 3 copies!

**Pull request merged**